### ISSUE TITLE
Nightly single user Che is broken

### ISSUE THREAD

**Issue by AndrienkoAleksandr** [eclipse-che/che#19403]
### Che version

 - [ ] latest
 - [ ] nightly
 - [ ] other: please specify

### Steps to reproduce

1. Install Eclipse Che in the minikube:
```bash
 chectl server:deploy -n eclipse-che -p minikube --installer operator
```

2. Patch che-cluster to enable single user:

```bash
kubectl patch checluster/eclipse-che -n eclipse-che --type=merge -p '{"spec":{"server": {"customCheProperties": {"CHE_MULTIUSER": "false"}}}}'
````

3. Open user dashboard.  See error:

![Screenshot from 2021-03-25 21-59-43](https://user-images.githubusercontent.com/6873095/112535864-70714300-8db5-11eb-8d88-abf0649e9fee.png)

### Runtime
   - [ ] kubernetes (include output of `kubectl version`)
   - [ ] Openshift (include output of `oc version`)
   - [ ] minikube (include output of `minikube version` and `kubectl version`)
   - [ ] minishift (include output of `minishift version` and `oc version`)
   - [ ] docker-desktop + K8S (include output of `docker version` and `kubectl version`)
   - [ ] other: (please specify)

### Installation method
   - [ ] chectl
     * provide a full command that was used to deploy Eclipse Che (including the output)
     * provide an output of `chectl version` command
   - [ ] OperatorHub
   - [ ] I don't know

### Environment
   - [ ] my computer
       - [ ] Windows
       - [ ] Linux
       - [ ] macOS
   - [ ] Cloud
       - [ ] Amazon
       - [ ] Azure
       - [ ] GCE
       - [ ] other (please specify)
   - [ ] Dev Sandbox (workspaces.openshift.com)
   - [ ] other: please specify

**Comment by manojsaxenatda** [comment]
I was just about to open the same issue.  I have tried 7.28.0 and 7.28.2

**Comment by sleshchenko** [comment]
According to https://github.com/eclipse/che/issues/19434#issuecomment-809634613
the fix should be backported to 7.28.x

**Comment by garyyang85** [comment]
Encounter this issue on 7.31.2, installed by operator on openshift. But currently only reproduced on Firefox browser. Chrome works good.
